### DISCUSSION TITLE
Plan for 2nd version

### DISCUSSION THREAD

**Discussion by SamYuan1990** [hyperledger/fabric-admin-sdk#122] [General]
in our 1st version we are going to cover most of golang features.
in our 2nd version what do you want to cover? please let us know.

**Comment by SamYuan1990** [comment]
1. Test coverage
2. Validation
3. Lint with unuse and dupl
4. from @denyeart , fabric-config repo as basic model?

**Reply by SamYuan1990** [reply]
@bestbeforetoday , @davidkhala , @channingduan

**Comment by bestbeforetoday** [comment]
I have seen at least one request for better documentation and usage examples. Just pointing people to the e2e test is not really ideal.

**Reply by SamYuan1990** [reply]
en. In my point of view, I hope user can learn from our e2e test case and build their own. Hence we can save some effort on document/example?
